npm/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "@criticalmanufacturing/sdk",
+    "version": "1.1.2",
+    "description": "Critical Manufacturing CLI SDK",
+    "author": "Critical Manufacturing, SA",
+    "keywords": [
+        "cmf"
+    ],
+    "license": "BSD-3-Clause",
+    "homepage": "https://github.com/criticalmanufacturing/portal-sdk#readme",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/criticalmanufacturing/portal-sdk.git"
+    },
+    "bugs": {
+        "url": "https://github.com/criticalmanufacturing/portal-sdk/issues"
+    },
+    "bin": {
+        "cmf-sdk": "./bin/cmf-sdk"
+    },
+    "scripts": {
+        "postinstall": "node install.js"
+    },
+    "os": [
+        "win32",
+        "linux"
+    ],
+    "cpu": [
+        "x64"
+    ],
+    "dependencies": {
+        "@criticalmanufacturing/cli": "*",
```

Comment:
why is the cli dependency needed?

Reply:
The idea was to treat cmf-sdk as a cli plugin, so when the sdk is installed the cli will also immediately be available.
But since it is not technically a dependency, it has been removed.